docs/styling/THEME.md

```diff
@@ -21,24 +21,21 @@ For more information on the Amadeus palettes you can refer to [Amadeus Color Gui
 
 ## Create your theme
 
-Find below the process to generate your own theme. RefX is used here as an example, you can replace it with any other theme.
+You can find below the process to generate your own theme.
 
-### Generate the RefX theme variables
+### Generate your own theme variables
```

Comment:
Note: there are other reference to refx and airlines in the examples and in this documentation.